pages/History.qml

```diff
@@ -633,17 +633,7 @@ Rectangle {
                                 MoneroComponents.TextPlain {
                                     font.family: MoneroComponents.Style.fontRegular.name
                                     font.pixelSize: 15
-                                    text: {
-                                        if (!isout) {
-                                            return qsTr("Received") + translationManager.emptyString;
-                                        }
-                                        const addressBookName = currentWallet ? currentWallet.addressBook.getDescription(address) : null;
-                                        if (!addressBookName)
-                                        {
-                                            return qsTr("Sent") + translationManager.emptyString;
-                                        }
-                                        return addressBookName;
-                                    }
+                                    text: isout ? qsTr("Sent") : qsTr("Received") + translationManager.emptyString
```

Comment:
`1 ? 2 : 3 + 4` == `2`

Reply:
i don't see what's wrong here

Reply:
add () around the ternary operator so that + emptyString gets applied to both

Reply:
+ emptyString is already being applied to both Sent and Received

Reply:
no, it only gets applied to Received due to + having precedence over the ternary operator

Reply:
in my wallet the Sent string is being translated normally. And I've already seen this type of `A ? B : C + translationManager.emptyString` many times in monero GUI code, always working fine.

Reply:
> in my wallet the Sent string is being translated normally.

Yes because `translationManager.emptyString` is technically not needed when using the ternary operator, not because the () are not needed.

> And I've already seen this type

Then it is wrong in these places too, would appreciate it if you could update it here, else it is wrong code that works by accident.

Reply:
My code is:
`isout ? qsTr("Sent") : qsTr("Received") + translationManager.emptyString`

Actually the computer reads this (but somehow translation is working):
`(isout ? qsTr("Sent") : qsTr("Received") + translationManager.emptyString)`

And you want me to change to this:
`(isout ? qsTr("Sent") : qsTr("Received")) + translationManager.emptyString`

Right?

Reply:
> but somehow translation is working

Due to technically `translationManager.emptyString` not being needed at all in this case. But it is better if we still add it.

> And you want me to change to this:

Yes.

Reply:
Why is translationManager.emptyString not needed in this case? I thought that I should add `+ translationManager.emptyString` at the end of every line that contains a `qsTr("");`, so that the string gets added to the translation.

Reply:
translationManager.emptyString is a workaround for a Qt bug, AFAIK it is only needed in cases like this:

```
text: amount
```

We add it to every string just to be sure.